x-pack/plugins/observability_solution/observability_ai_assistant_app/common/functions/changes/index.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FromSchema } from 'json-schema-to-ts';
+import { ChangePointType } from '@kbn/es-types/src';
+
+export const changesFunctionParameters = {
+  type: 'object',
+  properties: {
+    start: {
+      type: 'string',
+      description:
+        'The beginning of the time range, in datemath, like now-24h, or an ISO timestamp',
+    },
+    end: {
+      type: 'string',
+      description: 'The end of the time range, in datemath, like now, or an ISO timestamp',
+    },
+    logs: {
+      description: 'Analyze changes in log patterns',
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          name: {
+            type: 'string',
+            description: 'The name of this set of logs',
+          },
+          index: {
+            type: 'string',
+            description: 'The index or index pattern where to find the logs',
```

Comment:
How does the LLM know the logs index pattern?

Reply:
Yeah good question, maybe we should remove it entirely, and always use whatever we have in the advanced setting, or at least mention the default here in the description. WDYT?